### DISCUSSION TITLE
Should I transfer the repo for luayue definitions to the LuaCATS organisation?

### DISCUSSION THREAD

**Discussion by Frityet** [LuaLS/LLS-Addons#93] [General]
I am the creator and maintainer of [LuaYue definitions](https://github.com/Frityet/luayue-definitions), should I transfer it to the LuaCATS org?

**Comment by carsakiller** [comment]
The LuaCATS organization is intended to be a place where addons maintained by the community as a collective are housed. Essentially for projects without a maintainer that, instead of being abandoned, can ideally be kept alive by the community there.

**Reply by Frityet** [reply]
Got it! For now ill keep it seperate then
